### ISSUE TITLE
need SHTC3 driver for this to be useful

### ISSUE THREAD

**Issue by dstumbo** [Sensirion/python-i2c-stc#1]
In conversations with Reto in customer support, he said that in order to get reasonable accuracy from the SEK-STC31-Sensor kit, I need to use the temperature and humidity from the onboard SHTC3. He suggested using the sensirion python-i2c-sht driver or the adafruit python SHTC3 driver.

I cannot get the sensirion sht driver to work with the SHTC3. Should it? I am using the linux interface, not the SensorBridge. The adafruit python driver works, but it uses a different approach to the i2c bus, so I can't use them both at the same time.

Request (in order of preference)
1. example code using the linux transceiver to get readings from the SHTC3 on the SEK-STC31-Sensor
2. STC31 driver that works in the adafruit circuitpython environment
3. c code that will interface to both
4. some other suggestion that will allow me to use the SEK-STC31-sensor with the onboard SHTC3

Thanks!

**Comment by MBjoern** [comment]
Did you see our c drivers for sht and stc sensors?

https://github.com/Sensirion/embedded-i2c-stc3x
https://github.com/Sensirion/embedded-sht

Alternatively you should be able to combine the python drivers for STC3x and SHT, when getting a sensor bridge, which isn't that expensive after all.

In the mean time, we can look into possibilities to run the python code w/o Sensor Bridge. In theory, you'd need to replace the Sensor Bridge Transceiver calls with your own and should get it running from there. Did you already try this?

**Comment by dstumbo** [comment]
Hi Bjorn,

Thanks for getting back to me.

I did see the c code. My code is in python, so I was hoping for a python solution. I ended up using sensirion code to talk to the stc3x and Adafruit’s code to talk to the sht. I was concerned that the two different interfaces to the i2c bus might lead to problems, but in initial testing it seemed to work.

I guess it just seems odd to me that Sensirion would provide a python driver for 1/2 of the recommended sensor pair, so I thought I might be missing something.

Thanks,

Dave

Dave Stumbo
